Assets/UI/ToolTips/plottooltip.lua

```diff
@@ -531,7 +546,7 @@ function View(data:table, bIsUpdate:boolean)
       if (data.WonderType == nil) then
         table.insert(details, Locale.Lookup("LOC_TOOLTIP_PLOT_BUILDINGS_TEXT"));
       end
-			local greatWorksSection: table = {};
+      local greatWorksSection: table = {};
       for i, v in ipairs(data.BuildingNames) do
           if (data.WonderType == nil) then
```

Comment:
can you reindent this line (minus 2 spaces)

Reply:
Hey,
you want to have the line "local greatWorksSection" (new line 549) to have two spaces less?
But why? It´s on the same indentation as the lines before and after.
Or am I understanding something wrong? :)

Reply:
Ah ok - all fine. forget it, noticed u mean line 551 :)